doc/DESI_SPECTRO_DATA/NIGHT/EXPID/fiberassign-EXPID.rst

```diff
@@ -289,54 +289,54 @@ Required Data Table Columns
 ================================= ======= ===== ===================
 Name                              Type    Units Description
 ================================= ======= ===== ===================
```

Comment:
This table is malformed, because the width of the Units column does not match the actual content of the column.  Please make sure all table columns have the correct width.